### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by santisoler**
Add `po/es.po` file with Spanish translation for Docklike plugin.

### CODE REVIEW

**Comment by santisoler** [comment]
Thanks @nsz32 for this awesome plugin! Just wanted to add a minor contribution.

File: po/es.po [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+msgstr "Puntos"
+
+#: ../src/_dialogs.xml.h:10
+msgid "Rectangles"
+msgstr "Rectángulos"
+
+#: ../src/_dialogs.xml.h:11
+#, fuzzy
+msgid "Indicator orientation :"
+msgstr "Posición del indicador :"
```
Comment (santisoler) [inline po/es.po:111]: I translated this to something like `Indicator position` rather than `Indicator orientation`. I think that `position` sounds better, even for English. Maybe that's something that could be changed on a different PR.

Let me know what do you think.

Comment (nsz32) [reply po/es.po:111]: Haha, this is debatable :)

**Comment by nsz32** [comment]
Thanks for the translation to your great latin tongue !
Merci ! Soyons libres !

**Comment by santisoler** [comment]
Thanks @nsz32 for merging my contribution! Hope to be able to offer more help in the future.
